cli/cook/subcommands/kill.py

```diff
@@ -0,0 +1,109 @@
+from cook import http, colors
+from cook.querying import query, print_no_data
+from cook.util import strip_all, print_info
+
+
+def guard_against_duplicates(uuids, query_result):
+    """
+    Checks for UUIDs that are duplicated and throws if any are found (unlikely to happen in the wild, but it could)
+    """
+    if query_result['count'] == 1:
+        return
+
+    uuid_to_entries = {}
+    duplicate_uuids = []
+
+    def add(uuid, entity_type, cluster):
+        entry_map = {'type': entity_type, 'cluster_name': cluster}
+        if uuid in uuid_to_entries:
+            uuid_to_entries[uuid].append(entry_map)
+            duplicate_uuids.append(uuid)
+        else:
+            uuid_to_entries[uuid] = [entry_map]
+
+    for cluster_name, entities in query_result['clusters'].items():
+        jobs = entities['jobs']
+        for job in jobs:
+            add(job['uuid'], 'job', cluster_name)
+
+        instances = [i for j in entities['instances'] for i in j['instances'] if i['task_id'] in uuids]
+        for instance in instances:
+            add(instance['task_id'], 'job instance', cluster_name)
+
+        groups = entities['groups']
+        for group in groups:
+            add(group['uuid'], 'job group', cluster_name)
+
+    if len(duplicate_uuids) > 0:
+        messages = []
+        for duplicate_uuid in duplicate_uuids:
+            bullets = []
+            for entry in uuid_to_entries[duplicate_uuid]:
+                bullets.append(f'- as a {entry["type"]} on {entry["cluster_name"]}')
+            message = f'{duplicate_uuid} is duplicated:\n' + '\n'.join(bullets)
+            messages.append(message)
+        details = '\n\n'.join(messages)
+        message = f'Unable to kill due to duplicate UUIDs.\n\n{details}'
```

Comment:
"Refusing to kill" might be a more accurate description.

It would also be good to tell the user how to get around this issue (i.e., tell them to explicitly set the cluster where they want to kill the duplicated entity).

Reply:
Done and done